flink-cdc-cli/src/main/java/com/ververica/cdc/cli/CliFrontendOptions.java

```diff
@@ -52,12 +52,35 @@ public class CliFrontendOptions {
                     .desc("Use Flink MiniCluster to run the pipeline")
                     .build();
 
+    public static final Option SAVEPOINT_PATH_OPTION =
+            Option.builder("s")
+                    .longOpt("fromSavepoint")
+                    .hasArg(true)
+                    .desc(
+                            "Path to a savepoint to restore the job from (for example hdfs:///flink/savepoint-1537")
+                    .build();
+
+    public static final Option SAVEPOINT_RESTORE_MODE =
+            Option.builder("rm")
```

Comment:
`rm` is a dangerous abbreviation in CS which usually means `remove`, please avoid use this abbreviation